layers/+source-control/git/packages.el

```diff
@@ -362,6 +362,7 @@
         "C" 'forge-checkout-pullreq
         "b" 'forge-browse-topic
         "d" 'forge-delete-comment
```

Comment:
let's make the deletion "D".
in principle, destructive operations should be cubersome to enter.
then we can also use "d" to toggle draft.

Reply:
sounds good.